specifyweb/frontend/js_src/lib/components/TreeView/Actions.tsx

```diff
@@ -234,25 +239,33 @@ function EditRecordDialog<SCHEMA extends AnyTree>({
         node.set('parent', parentNode.url());
       }
       return node;
-    }, [nodeId, tableName, addNew])
+    }, [isOpen, nodeId, tableName, addNew])
   );
 
   return (
-    <ResourceLink
-      component={Link.Icon}
-      props={{
-        'aria-disabled': disabled,
-        icon: addNew ? 'plus' : 'pencil',
-        title: label,
-      }}
-      resource={resource}
-      resourceView={{
-        dialog: 'nonModal',
-        onAdd: isRoot ? undefined : setResource,
-        onDeleted: handleRefresh,
-        onSaved: handleRefresh,
-      }}
-    />
+    <>
+      <Button.Icon
```

Comment:
can you turn this into Link.Icon like it was before?
and then in onClick do `(event): void => { event.preventDefault(); handleToggle(); }`

this way, users can still right click on the link to open it in new tab, but on regular click it acts like a button

Reply:
Thought about it, but then you have another bug: User clicks on edit. Clicks on Clone (resource state gets set to clone resource). Clicks on Close button. Resource is still the cloned resource. So, the next time they click on Edit it will show cloned resource (until they click somewhere else and then back on current node)

This bug doesn't happen in production because dependency of callback in live state includes  `isOpen`